### PR TITLE
Intelligently splits flac file such that no individual file exceeds 100mb

### PR DESCRIPTION
**Pull request by yoninachmany**
r @sandlerben @roornelas 

IBM only accepts files that are 100mb in size -To catch all the words, repeats a 5 s section. -SplitFlacFile converts file to Wav, gets number of chunks including redundant 5 seconds using a helper function, and extracts audio segments of a given length using a derivation (see comment) and another helper function
in process of mock testing!

### CODE REVIEW

**Comment by sandlerben** [comment]
@yoninachmany don't mock test this for now. We are going to change how we do mocks soon.

**Comment by sandlerben** [comment]
@yoninachmany can you fix the conflicts

**Comment by yoninachmany** [comment]
Done

File: templates/form.html [hunk 1]
```diff
@@ -81,6 +81,14 @@ <h2 class="ui blue image header">
             <input type="text" id="words" name="words" placeholder="Search words" multiple>
           </div>
         </div>
+        <div class="grouped fields">
```
Comment (sandlerben) [inline templates/form.html:84]: What is this?

**Comment by sandlerben** [comment]
Can you make a new PR off of master and make sure to merge in master. I'm confused why the diff for this PR doesn't show any changes having to do with splitting the file.

**Pull request closed without merging**